pages/clubs/ClubForm.jsx

```diff
@@ -83,134 +114,41 @@ var ClubForm = React.createClass({
-    }
-
-    var buttons = [];
-
-    if (this.state.currentStep > 0) {
-      buttons.push(
-        <button key={'back'} className="back btn" disabled={this.state.submitting} onClick={!this.state.submitting && this.prevStep}>Back</button>
-      );
-    }
-
-    var buttonClass = 'btn';
-    var buttonLabel = 'Next';
-
-    if (this.state.currentStep === 1) {
-      buttonLabel = "Submit";
-    }
-    if (this.state.submitting) {
-      buttonClass += ' submitting';
-      buttonLabel = 'Submitting...';
-    }
-
-    buttons.push(
-      <button key={'continue'} className={buttonClass} disabled={this.state.submitting} onClick={!this.state.submitting && this.nextStep}>{buttonLabel}</button>
-    );
-
-    return (
-      <div key="buttons" className="proceed">
-        <div>{buttons}</div>
-        { (this.state.currentStep < 2) ? <ProgressBar value={this.state.progress}/> : null }
-      </div>
-    );
+  updateProgress: function(progress) {
+    console.log(progress);
   },
 
-  updateProgress: function() {
-    var r1 = this.refs.step1;
-    var r2 = this.refs.step2;
-
-    if (!r1 || !r2) {
-      return 0;
-    }
-
-    var total = r1.getTotal() + r2.getTotal();
-    var filled = r1.getFilled() + r2.getFilled();
-    var percent = (100*filled/total) | 0;
-
-    this.setState({ progress: percent });
-  },
-
-  prevStep: function() {
-    this.setState({
-      currentStep: Math.max(this.state.currentStep - 1, 0)
-    });
-  },
-
-  nextStep: function() {
-    var refname = 'step' + (this.state.currentStep+1);
-    var curRef = this.refs[refname];
-    var validates = curRef.validates();
-
-    if (validates) {
-      var nextStep = Math.min(this.state.currentStep + 1, 2);
-      var goToNext = function() {
-        this.setState({ currentStep: nextStep });
-      }.bind(this);
-
-      if (this.state.currentStep === 1) {
-        this.submitForm(goToNext);
-      } else {
-        goToNext();
-      }
-    }
-  },
-
-  submitForm: function(next) {
+  submitForm: function(formData) {
     var teachAPI = this.props.teachAPI;
-
-    // new form data as object
-
-    var clubState = this.getClubData();
+    var clubState = formData;
 
     clubState.longitude = clubState.location.longitude;
     clubState.latitude = clubState.location.latitude;
     clubState.location = clubState.location.location;
 
-    // send to Teach-API and wait for response via the callback
-    var networkHandler = this.handleNetworkResult;
-
     this.setState({
       submitting: true,
-      step: this.STEP_WAIT_FOR_NETWORK,
       networkError: false,
-    }, function() {
-      teachAPI.addClub(clubState, function(err, data) {
-        networkHandler(err, data, next);
+      step: STEP_SENDING_RESULTS
+    }, () => {
+      console.log(clubState);
+
+//      teachAPI.addClub(clubState, (err, data) => {
+//        if (err) {
+//          this.setState({
+//            networkError: !!err,
+//            result: err ? null : data
+//          })
+//        } else {
```

Comment:
Not in the slightest - this is commented off so it doesn't try to send the data to the Teach API for PR purposes,  which the real code definitely needs to do =)

But requires 3 additional services to run... =(